src/app/_components/Footer/Footer.js

```diff
@@ -1,5 +1,101 @@
-import React from 'react';
+"use client";
```

Comment:
🔴 why is it client component? i don't see any client specific features used here

Reply:
Deleted.